docs/spl-docs/rex-command.md

```diff
@@ -0,0 +1,104 @@
+# rex
+
+## Description
+
+Use this command to either extract fields using regular expression named groups, or replace or substitute characters in a field using sed expressions.
+
+The rex command matches the value of the specified field against the unanchored regular expression and extracts the named groups into fields of the corresponding names.
+
+## Syntax
+
+The required syntax is in **bold**.
+
+**rex** \
+**[field=\<field>]** \
+**\<regex-expression>**
+
+## Required Arguments
+
+### field
+
+**Syntax:** `field=<field>` \
+**Description:** The field that you want to extract information from.
+
+### regex-expression
+
+**Syntax:** `"<string>"` \
+**Description:** The PCRE regular expression that defines the information to match and extract from the specified field.
+
+### Examples
+
+### 1. Extract major and minor version number from an app version.
+
+This search extracts the major and minor version numbers from the `app_version` field using named groups and displays them as separate fields.
+
+```
+... | rex field=app_version "(?<major>\d+)\.(?<minor>\d).*"  
+| fields app_version, major, minor
+```
```

Comment:
nit: should `(?<minor>\d)` be `(?<minor>\d+)` instead, so that the `minor` can have multiple digits? Or if `\d` matches multiple digits, maybe you don't need the `+` in `major`?

Reply:
Yes, the `+` is needed to match multiple digits.